save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -62,10 +62,35 @@ external interface ProjectExecutionRouteProps : PropsWithChildren {
     var currentUserInfo: UserInfo?
 }
 
+/**
+ * State with role
+ */
+external interface StateWithRole : State {
```

Comment:
Why is it defined only here? I mean, it can be used all over the project

Reply:
I tried to split a huge external inteface to several with fields are related to a single functionality.
And yes, it can be used in another places -- will extract to common file